components/provisioner/internal/provisioning/input_converter.go

```diff
@@ -74,7 +78,7 @@ func (c converter) ProvisioningInputToCluster(runtimeID string, input gqlschema.
 	}, nil
 }
 
-func (c converter) gardenerConfigFromInput(runtimeID string, input *gqlschema.GardenerConfigInput) (model.GardenerConfig, apperrors.AppError) {
+func (c converter) gardenerConfigFromInput(runtimeID string, input *gqlschema.GardenerConfigInput, tillerYaml string) (model.GardenerConfig, apperrors.AppError) {
```

Comment:
What about just passing boolean if the privileged containers should be allowed?

Reply:
I was thinking about that and I just couldn't decide on which layer of abstraction should the shouldAllowPrivilegedContainers mechanism appear :D If you say it'll be cleaner I change it